### PR TITLE
notebook installation help

### PR DESCRIPTION
**Pull request by CerenB**

### CODE REVIEW

File: notebooks/README.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 # README
 
-1. Make sure that you have Octave installed. 
+1. Make sure that you have Octave installed.
 2. If you have Conda/Jupyter/pip installed, go to step 4.
+Check is Conda installed properly by `conda list` typing into your terminal
```
Comment (Remi-Gau) [inline notebooks/README.md:5]:
```suggestion
Check if Conda installed properly by `conda list` typing into your terminal
```

**Review by Remi-Gau — CHANGES_REQUESTED** [review]
just a typo to fix

File: notebooks/README.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
 3. Download the [Anaconda Installer](https://www.anaconda.com/products/individual) and install it.
-4. Install [Octave kernel](https://pypi.org/project/octave-kernel/): 
+If using miniconda, run `conda install jupyter` to download and install the Jupyter Notebook package.
+4. Install [Octave kernel](https://pypi.org/project/octave-kernel/):
```
Comment (marcobarilari) [inline notebooks/README.md:8]: I will try all these steps in the brand 'new' old mac to see if it works and if there are faster ways like installing stuff e.g. via homebrew.

Related to this, I fear that these installing steps can be done in different (and faster) ways depending on the OS. What do you think?

Worth to point to different ways depending on the OS?

Comment (CerenB) [reply notebooks/README.md:8]: I think anaconda/conda provide very good support on the issue. We should not spend time on it thaaaat much. 
I've tried these with my mac, thus I added the tiny suggestions.

Comment (Remi-Gau) [reply notebooks/README.md:8]: Best way is to point to other doc where they have that nailed already but I agree that for OS specific info is nice.

I think I had taken that from Agah's repo from the OHBM hackathon.

Maybe we have some more install info on the hackathon website.

Comment (Remi-Gau) [reply notebooks/README.md:8]: But in general I would have an approach of "some doc is better than none and update it when someone points out it is not clear".

File: notebooks/README.md (hunk 1 above)
Comment (marcobarilari) [inline notebooks/README.md:5]: btw, I have the 'Anaconda navigator app' installed but if I tyoe `conda list` the terminal says tha does not know the conda command

Comment (marcobarilari) [reply notebooks/README.md:5]: ok somehow re installing Anaconda worked

**Comment by marcobarilari** [comment]
I think this one is ready to be merged, what you think?

File: notebooks/README.md [hunk 3]
```diff
@@ -1,9 +1,11 @@
 # README
 
-1. Make sure that you have Octave installed. 
+1. Make sure that you have Octave installed.
 2. If you have Conda/Jupyter/pip installed, go to step 4.
+Check if Conda installed properly by `conda list` typing into your terminal
```
Comment (marcobarilari) [inline notebooks/README.md:5]:
```suggestion
Check if Conda is installed properly by typing `conda list` into your terminal
```

**Review by marcobarilari — APPROVED** [review]

**Review by Remi-Gau — APPROVED** [review]

**Comment by Remi-Gau** [comment]
@CerenB 

I will reopen this one to have as a PR on `dev` and not on `master`. Sorry about that. :-)

**Pull request closed without merging**